src/stores/DeviceStore.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const alt = require('../alt');
+
+const DeviceActions = require('../actions/DeviceActions.js');
+
+class DeviceStore {
+  constructor() {
+
+    this.bindActions(DeviceActions);
```

Comment:
I prefer to be explicit with the actions you are binding.  Binding to all can cause extra overhead that we don't want to have